NEWS.txt

```diff
@@ -1,27 +1,66 @@
 ﻿           libSBML NEWS -- History of user-visible changes
 
 ====================================================================
-5.18.1 (Current svn)
+5.19.0
 ====================================================================
 
-STABLE RELEASE
+********************************************************************
+
+        The libSBML source code has moved to GitHub.
+            (https://github.com/sbmlteam/libsbml). 
+
+The stable branch reflect the latest released code. The development 
+branch is the main branch and we encourage users to create and 
+submit pull requests against this branch. We are no longer 
+maintaining a separate repository for experimental code for as yet 
+unfinished packages. 
+
+Please use the GitHub issue tracker to file any reports. We will gradually 
+move away from SourceForge, but for this release code will be 
+available from both SourceForge and GitHub. 
+
```

Comment:
maybe encourage people on filing pull requests for changes on the repo directly? It worked wonders for the people in my lab.

Reply:
done